ugandaemr_mobile/drug-administration.md

```diff
@@ -0,0 +1,14 @@
+# Drug Dispensing
+
+1. The screen belows shows how to administer drug, each patients current Regimen is displayed as shown in the figure below.
+
+   ![Drug Screen](.gitbook/assets/img3.jpg)
+
+4.Tap on the drug or combination from the Drug administration tables as shown below to enter pills dispensed and duration in days. ![Drug administration Screen](.gitbook/assets/img4.jpg)
```

Comment:
@solemabrothers Number the items in this list properly and start sentences with Capital Letters

Reply:
Done